src/main/java/net/minecraftforge/registries/tags/IReverseTag.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.registries.tags;
+
+import net.minecraft.tags.TagKey;
+
+import java.util.stream.Stream;
+
+public interface IReverseTag<V>
+{
+    Stream<TagKey<V>> getTagKeyStream();
```

Comment:
As there is no other getter and it is obvious this is a stream due to the return type I feel like a better name for this may just be `getTagKeys`